app/views/dashboard/_discovery_widget_host_list.html.erb

```diff
@@ -1,16 +1,16 @@
 <table class="table table-striped ellipsis">
   <thead>
+  <th></th>
   <th><%= _('Host') %></th>
   <th><%= _('Model') %></th>
-  <th><%= _('IP Address') %></th>
   <th><%= _('CPUs') %></th>
   <th><%= _('Memory') %></th>
   </thead>
   <tbody>
   <% hosts.each do |host| %>
     <tr>
-      <%= render :partial => "discovered_hosts/discovered_host", :locals => {
```

Comment:
I think we can delete this whole file completely. It is not used anymore. I mean the `discovered_hosts/discovered_host` partial.